src/frontend/src/styles/main.css

```diff
@@ -698,6 +701,12 @@ a:hover,
 .c-icon--ok {
   --onBgc: var(--rc-positive);
 }
+.c-icon--info__flipped {
```

Comment:
`__` would be a child in BEM. I would create a separate class called `.c-icon--inverted` that combines with any icon class. But since it looks like info is always flipped, it might not need a modifier? So just `c-icon--info`, makes sense?